pkgs/applications/emulators/ccemux/default.nix

```diff
@@ -64,5 +64,6 @@ stdenv.mkDerivation rec {
     sourceProvenance = with sourceTypes; [ binaryBytecode ];
     license = licenses.mit;
     maintainers = with maintainers; [ CrazedProgrammer viluon ];
+    broken = true; # download of CCEmuX-cct.jar fails hash validation
```

Comment:
can we update hash ?

Reply:
Since I don't use this software I am not qualified to decide if that's safe.

Usually the best course here is to mark the package as `broken` and if nobody steps up, let the package get garbage-collected.  If somebody does step up to audit the mysteriously-changed binary jarfile, and wants to take the reputational risk upon themselves, more power to them!

Reply:
maybe @viluon can have a look ( last committer ), otherwise I agree with your assessment

Reply:
I agree with @amjoseph-nixpkgs . My nitpick is to open a tracking issue and reference it. So discussions of fixing that package happens in that issue.

Reply:
Upstream doesn't do version control on binaries AFAIK. The maintainer will have to be on top of version bumps to avoid this issue from happening. I don't use Nix, so I don't know if this is an option (probably isn't), but it may also be necessary to skip checking hashes to avoid breakages between updates.

Reply:
Nah, see CCEmuX/CCEmuX#167.